src/core/thread/mle_tlvs.cpp

```diff
@@ -84,5 +84,42 @@ void ConnectivityTlv::IncrementLinkQuality(LinkQuality aLinkQuality)
     }
 }
 
+int8_t ConnectivityTlv::GetParentPriority(void) const
+{
+    int8_t priority = kParentPriorityUnspecified;
+
+    switch (mFlags & kFlagsParentPriorityMask)
+    {
+    case kFlagsPriorityHigh:
+        priority = kParentPriorityHigh;
+        break;
+    case kFlagsPriorityMedium:
+        priority = kParentPriorityMedium;
+        break;
+    case kFlagsPriorityLow:
+        priority = kParentPriorityLow;
+    default:
+        break;
+    }
+
+    return priority;
+}
+
+void ConnectivityTlv::SetParentPriority(int8_t aPriority)
+{
+    if (aPriority > 0)
+    {
+        mFlags = kFlagsPriorityHigh;
```

Comment:
I know the other bits are reserved, so this works for now. If/when we do decide to use those bits for other purposes, we'll need to update this as well.

Reply:
Yes. We do.